src/Main.ump

```diff
@@ -80,6 +73,41 @@ class DirectoryConverter {
 	    	System.out.println("No directory...");
 	    }
 	}
+	
+	private void saveTemplates(Path saveTo) {
+		saveTo.toFile().mkdirs();
+		
+		// Save the templates first
+		for (Map.Entry<String, TemplateConverter> fileNameTemplate : templates.entrySet()) {
+			String name = fileNameTemplate.getKey();
+			
+			try {
+				//System.out.println("Save to " + saveTo.resolve(name + ".ump"));
+				fileNameTemplate.getValue().save(saveTo.resolve(name + ".ump"), name, mainClassToStore);
+			} catch (FileNotFoundException e) {
+				// TODO Auto-generated catch block
+				e.printStackTrace();
```

Comment:
I realized I may have confused you and @bitwizeshift. Instead of just printing, rethrow checked exceptions in their unchecked counter part. It stops silent failures from happening. 

Oops, I should sleep more, too.